docs/_markbind/layouts/default.md

```diff
@@ -0,0 +1,79 @@
+<head-bottom>
+  <link rel="stylesheet" href="{{ baseUrl }}/stylesheets/main.css">
+</head-bottom>
+
+<header fixed>
+  <navbar placement="top" type="inverse">
+    <a slot="brand" href="{{ baseUrl }}/index.html" title="Home" class="navbar-brand">
+      <pic src="{{ baseUrl }}/teammateslogo.png" width="150" alt="Logo" caption=""/>
+      <span style="font-style:italic;font-size:small">[dev docs]</span>
+    </a>
+    <li><a href="{{ baseUrl }}/index.html" class="nav-link">Home</a></li>
+    <li><a href="{{ baseUrl }}/contributing-doc.html" class="nav-link">Contributing</a></li>
+    <li><a href="{{ baseUrl }}/about.html" class="nav-link">About</a></li>
+    <li><a href="https://teammatesv4.appspot.com/" target="_blank" class="nav-link">Product Website <md>:glyphicon-share-alt:</md></a></li>
+    <li><a href="https://github.com/TEAMMATES/teammates" target="_blank" class="nav-link"><md>:fab-github:</md></a></li>
+  <li slot="right">
+    <form class="navbar-form">
+      <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback" menu-align-right></searchbar>
+    </form>
+  </li>
+  </navbar>
+</header>
+
+<div id="flex-body">
+  <nav id="site-nav" class="fixed-header-padding">
+    <div class="site-nav-top">
+      <div class="font-weight-bold mb-2" style="font-size: 1.25rem;">Table of Contents</div>
```

Comment:
I really don't think "Table of Contents" value-add here. Everything in the sidebar is already implied as one in the context of the site's layout.